src/main/java/codeu/model/data/ConversationStatBot.java

```diff
@@ -30,7 +29,7 @@
   */
   public ConversationStatBot() {
     super(
-        UUID.randomUUID(), 
+        UUID.nameUUIDFromBytes("ConversationStatBot".getBytes()), 
```

Comment:
One more piece that calls for `Bot` to be a base class instead of an interface. :)

Reply:
Part of me wants to update this but part of me also wants to work on new features. What do you guys think?

Reply:
It's not worth it if it's a big change, I thought it was just moving this inside the `addUser` method. If not, never mind. :)

Reply:
Ah, I did that haha. I was referring to the `Bot` base class instead of the interface.